### PR TITLE
added outgoing registration fields

### PR DESCRIPTION
**Pull request by Ivanov-Anton**
additional fields for outgoing registration:
`

- last_reply_contacts
- last_request_contact
- pop_id
`
tring to solve issue https://github.com/yeti-switch/yeti-web/issues/596

### CODE REVIEW

File: spec/features/realtime_data/outgoing_resgistrations/index_outgoing_registrations_spec.rb [hunk 1]
```diff
@@ -6,7 +6,9 @@
   include_context :login_as_admin
 
   let!(:node) { FactoryGirl.create(:node) }
+  let!(:pop) { FactoryGirl.create(:pop) }
   let(:record_attributes) { FactoryGirl.attributes_for(:outgoing_registration, :filled) }
```
Comment (senid231) [inline spec/features/realtime_data/outgoing_resgistrations/index_outgoing_registrations_spec.rb:10]: add new attributes to `factory :outgoing_registration`

File: spec/features/realtime_data/outgoing_resgistrations/index_outgoing_registrations_spec.rb [hunk 2]
```diff
@@ -17,4 +19,18 @@
...
+      visit outgoing_registrations_path(q: { node_id_eq: node.id })
+    end
+
+    it 'has record' do
+      expect(page).to have_css('.col-id', text: record_filled[:id])
+      expect(page).to have_css('.col-last_reply_contacts')
+      expect(page).to have_css('.col-last_request_contact')
+      expect(page).to have_css('.col-pop', text: pop.name)
+    end
+  end
```
Comment (senid231) [inline spec/features/realtime_data/outgoing_resgistrations/index_outgoing_registrations_spec.rb:35]: you don't need to add `before` in this context.
`before` from parent `context/describe` will be executed in nested context

in this context you need to override `record_attributes` and check that additional fields are shown correctly in table

like this
```ruby
  context 'with additional fields' do
    let(:record_attributes) do
      attributes_for(:outgoing_registration, :filled, pop: pop, last_reply_contacts: 'foo', last_request_contact: 'bar')
    end

    it 'has record' do
      expect(page).to have_css('.col-id', text: record_attributes[:id])
      expect(page).to have_css('.col-last_reply_contacts', text: record_attributes[:last_reply_contacts])
      expect(page).to have_css('.col-last_request_contact', text: record_attributes[:last_request_contact])
      expect(page).to have_css('.col-pop', text: pop.name)
    end
  end
```

Comment (Ivanov-Anton) [reply spec/features/realtime_data/outgoing_resgistrations/index_outgoing_registrations_spec.rb:35]: Should I check the value of the added fields?

Comment (senid231) [reply spec/features/realtime_data/outgoing_resgistrations/index_outgoing_registrations_spec.rb:35]: yes

Comment (Ivanov-Anton) [reply spec/features/realtime_data/outgoing_resgistrations/index_outgoing_registrations_spec.rb:27]: Can't fount  pop in table and in record_attributes Hash missing also ()
part from output error rspec:
`but there were no matches. Also found "pop", ""`
byebug:
record_attributes.key?(:pop_id) #=> false

Comment (Ivanov-Anton) [reply spec/features/realtime_data/outgoing_resgistrations/index_outgoing_registrations_spec.rb:31]:
```
def initialize(nodes, pops, params = {})
  @params = clean_search_params(params).with_indifferent_access
  @errors = []
  set_nodes(nodes)
  set_pops(pops)
end
  
```

Comment (Ivanov-Anton) [reply spec/features/realtime_data/outgoing_resgistrations/index_outgoing_registrations_spec.rb:31]: Maybe I should add parameter for initialize OutgoingRegistrations

**Comment by senid231** [comment]
CI passes
https://travis-ci.org/github/yeti-switch/yeti-web/builds/689769280

but here it shows like pending. Looks like bug on github

**Pull request merged**